### ISSUE TITLE
[BUG] Regression in v0.0.69

### ISSUE THREAD

**Issue by themadmrj** [jhomlala/betterplayer#526]
The last update possibly caused a regression on iOS (it works on Android). When I try to play a local video (fetched with gallery), it fails in the latest version (works in 0.0.68).

```
Unhandled Exception: Invalid argument(s): Passed file doesn't exists.
#0      BetterPlayerController._setupDataSource
package:better_player/…/core/better_player_controller.dart:469
#1      BetterPlayerController.setupDataSource
package:better_player/…/core/better_player_controller.dart:269
```

The url looks like this: `file:///var/mobile/Media/DCIM/100APPLE/IMG_0075.MOV`. Did something change in the required configs? When I change to 0.0.68, the same file is played without any issue.

**Comment by themadmrj** [comment]
Is it possible this fails, because those urls are links provided by `AVAsset` and not a direct link to the file itself? The way it works locally is that for videos from the users `Photos` app, `AVPlayer` knows how to interpret this sort of url and stream read from it directly and I think that's why it worked in the previous version without the check.

**Comment by levent-kantaroglu** [comment]
Same situation with only 0.0.69. I solved this by downgrading 0.0.68.

I checked, the file exists. And according to controller `playing:true`, but not playing and `isVideoInitialized:false`

**Comment by jhomlala** [comment]
Fixed in 0.0.70.
